application/src/main/java/org/togetherjava/tjbot/commands/tags/TagsCommand.java

```diff
@@ -41,12 +45,24 @@ public TagsCommand(TagSystem tagSystem) {
 
     @Override
     public void onSlashCommand(@NotNull SlashCommandEvent event) {
-        // TODO A list might be better than comma separated, which is hard to read
+
```

Comment:
Why the empty line here?

Reply:
Though it would be more readable if i create some empty space

Reply:
Fair. Up to you. Its just unusual to have an empty line at the beginning of a method. Never seen that before.